test/tests/selftest/pbs_dshutils_tests.py

```diff
@@ -86,3 +89,91 @@ def test_pbs_env(self):
         self.du.unset_pbs_environment(environ=['pbs_foo'])
         environ = self.du.parse_pbs_environment()
         self.assertNotIn('pbs_foo', environ, msg)
+
+    def check_access(self, path, mode=0o755, user=None, group=None, host=None):
+        """
+        Helper function to check user, group and mode of given path
+        """
+        if host is None:
+            result = os.stat(path)
+            m = result.st_mode
+            u = result.st_uid
+            g = result.st_gid
+
+        else:
+            py = self.du.which(host, 'python3')
+            c = '"import os;s = os.stat(\'' + path + '\');'
+            c += 'print((s.st_uid,s.st_gid,s.st_mode))"'
+            cmd = [py, '-c', c]
+            runas = self.du.get_current_user()
+            ret = self.du.run_cmd(host, cmd, sudo=True, runas=runas)
+            self.assertEqual(ret['rc'], 0)
+            res = ret['out'][0]
+            u, g, m = [int(v) for v in
+                       res.replace('(', '').replace(')', '').split(', ')]
```

Comment:
Since you're the one doing the print, and you're just going to get rid of the (), why print them in the first place?